schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT NOT NULL,
```

Comment:
- [ optional] you can make your ID column not null and auto-incremented instead of setting it your self

for example.... `id int NOT NULL IDENTITY(1,1)`

Reply:
Oh, that's nice! Thank you (: